### DISCUSSION TITLE
Provide attribute for gltf bin file source link because of different directory than gltf file

### DISCUSSION THREAD

**Discussion by felixhaeberle** [google/model-viewer#2241] [Q&A]
### Description

The bin file is always loaded from within the folder where the gltf file lives in, but there are cases where you don't want to store both files in the same directory, or, CMS and other content management platforms are changing the source directory of the gltf file on the first load to a cache link or something. 

Then, the bin file can't be loaded from this directory, because the link and destination has changed. 

It's not the best way to only assume where a file is and therefore it would be a good thing to only use the assumed destination as fallback and provide a `bin-src` where people can specify their bin source like `ios-src` – otherwise it defaults to the assumed destination, the gltf source directory. 

#### Live Demo
https://glitch.com/edit/#!/elated-melodic-hotel

I removed the bin file that the error occurs, but can't upload an assets in anther folder in glitch.

**Comment by elalish** [comment]
Yes, I think this is why single-file GLBs tend to be preferred. Is there a reason you need unpacked glTFs instead?

**Comment by GoelBiju** [comment]
@elalish I was going to say the same since some glTF assets still provide a bin file (e.g. the now deprecated Poly website) and it would be useful to have this included.

**Comment by elalish** [comment]
I believe this can be accomplished using our [mapURLs](https://modelviewer.dev/docs/#loading-staticMethods) method. Can someone verify if this works for your use case?
